runtime/wasm/src/host.rs

```diff
@@ -727,6 +735,9 @@ impl PartialEq for RuntimeHost {
             data_source_call_handlers,
             data_source_block_handlers,
             host_exports,
+
+            // Data sources created at different blocks can be considered duplicated.
```

Comment:
I don't understand this comment - wouldn't two `RuntimeHost` that are otherwise `eq` but differ in the creation block indicate an error? Not sure if we should assert that here, but maybe the comment should state that the rest of the code makes sure that that doesn't happen.

Reply:
I've reworded the comment.